### DISCUSSION TITLE
Game development 🐇

### DISCUSSION THREAD

**Discussion by AndreaGriffiths11** [community/community#22380] [New to GitHub]
<aside class="onebox twitterstatus" data-onebox-src="https://twitter.com/GitHubCommunity/status/1445928253789724681?s=20">
  <header class="source">

      <a href="https://twitter.com/GitHubCommunity/status/1445928253789724681?s=20" target="_blank" rel="noopener">twitter.com</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/20666190/181070662-ef0da042-de3a-4c1e-9b54-817fdeacfdcb.jpeg" class="thumbnail onebox-avatar" width="400" height="400">
<h4><a href="https://twitter.com/GitHubCommunity/status/1445928253789724681?s=20" target="_blank" rel="noopener">GitHub Community</a></h4>
<div class="twitter-screen-name"><a href="https://twitter.com/GitHubCommunity/status/1445928253789724681?s=20" target="_blank" rel="noopener">@GitHubCommunity</a></div>

<div class="tweet">
  Ready, set, game! Join us at 7PM SF for an hour discussing all things game development. We’ll be joined by special guests @MishManners @end3r &amp; @t3dotgg 

Drop your questions for our panel in this tweet, or request to speak live!
#gamedevelopment 

https://t.co/7rjnu19B1I
</div>

<div class="date">
  <a href="https://twitter.com/GitHubCommunity/status/1445928253789724681?s=20" class="timestamp" target="_blank" rel="noopener">6:45 PM - 6 Oct 2021</a>

    <span class="like">
      <svg viewbox="0 0 512 512" width="14px" height="16px" aria-hidden="true">
        <path d="M462.3 62.6C407.5 15.9 326 24.3 275.7 76.2L256 96.5l-19.7-20.3C186.1 24.3 104.5 15.9 49.7 62.6c-62.8 53.6-66.1 149.8-9.9 207.9l193.5 199.8c12.5 12.9 32.8 12.9 45.3 0l193.5-199.8c56.3-58.1 53-154.3-9.8-207.9z"></path>
      </svg>
      3
    </span>

    <span class="retweet">
      <svg viewbox="0 0 640 512" width="14px" height="16px" aria-hidden="true">
        <path d="M629.657 343.598L528.971 444.284c-9.373 9.372-24.568 9.372-33.941 0L394.343 343.598c-9.373-9.373-9.373-24.569 0-33.941l10.823-10.823c9.562-9.562 25.133-9.34 34.419.492L480 342.118V160H292.451a24.005 24.005 0 0 1-16.971-7.029l-16-16C244.361 121.851 255.069 96 276.451 96H520c13.255 0 24 10.745 24 24v222.118l40.416-42.792c9.285-9.831 24.856-10.054 34.419-.492l10.823 10.823c9.372 9.372 9.372 24.569-.001 33.941zm-265.138 15.431A23.999 23.999 0 0 0 347.548 352H160V169.881l40.416 42.792c9.286 9.831 24.856 10.054 34.419.491l10.822-10.822c9.373-9.373 9.373-24.569 0-33.941L144.971 67.716c-9.373-9.373-24.569-9.373-33.941 0L10.343 168.402c-9.373 9.373-9.373 24.569 0 33.941l10.822 10.822c9.562 9.562 25.133 9.34 34.419-.491L96 169.881V392c0 13.255 10.745 24 24 24h243.549c21.382 0 32.09-25.851 16.971-40.971l-16.001-16z"></path>
      </svg>
      6
    </span>
</div>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>Ready, set, game! Join us at 7PM SF for an hour discussing all things game development. We’ll be joined by special guests <a href="https://www.linkedin.com/in/ACoAAA8W5xsBxNBS8J7LtQcUTKTZIOd73e7Tjvo">Michelle Mannering</a>, <span class="mention">@endr3</span> Andrzej Mazur and <span class="mention">@theobr</span> Theo Browne.</p>
<p>Drop your questions for our panel in the tweet, or request to speak live!<br>
<a href="https://www.linkedin.com/feed/hashtag/?keywords=gamedevelopment&amp;highlightedUpdateUrns=urn%3Ali%3Aactivity%3A6851695784983965696">#gamedevelopment</a><br>
<a href="https://www.linkedin.com/feed/hashtag/?keywords=easteregg&amp;highlightedUpdateUrns=urn%3Ali%3Aactivity%3A6851695784983965696">#easteregg</a></p>

**Comment by AndreaGriffiths11** [comment]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/2/e/2ecf2481fea4b354d4d5ab73d394942cf73b434c.jpeg" data-download-href="/uploads/short-url/6G5LpfnRBIKJIPsXyaYKAz1IH5q.jpeg?dl=1" title="image"><img src="https://user-images.githubusercontent.com/20666190/181070665-f23a0140-066a-4020-a2ff-c00e2499d1eb.jpeg" alt="image" data-base62-sha1="6G5LpfnRBIKJIPsXyaYKAz1IH5q" width="690" height="220" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/2/e/2ecf2481fea4b354d4d5ab73d394942cf73b434c_2_690x220.jpeg, https://aws1.discourse-cdn.com/github/original/3X/2/e/2ecf2481fea4b354d4d5ab73d394942cf73b434c.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/3X/2/e/2ecf2481fea4b354d4d5ab73d394942cf73b434c.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/2/e/2ecf2481fea4b354d4d5ab73d394942cf73b434c_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">940×300 76.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by whuang8** [comment, marked as answer]
<p>A tennis game created in October 1958 by William Higinbotham!</p>
<p>Twitter handle: whuang8</p>

**Comment by Tycee-Eecyt** [comment]
<p>Hey, just think I found the post, <span class="mention">@Klikxp99</span> is my twitter handle. :wave:</p>

**Comment by AndreaGriffiths11** [comment]
<p><img src="https://user-images.githubusercontent.com/20666190/181070669-0f906db3-3d8b-4c33-bb00-36e509614ffc.gif" alt="giphy" data-base62-sha1="yots2cFr8obJLFZycS8NNXosyKu" width="480" height="270" class="animated"></p>

**Comment by UWA-Ally** [comment]
<p>UWA Technology, which continues to provide product performance solutions and technical support for the game and VR industries, has cumulatively supported more than 8,600 products in the past six years. At the same time, adhering to the unique business philosophy of “altruism”, UWA is also continuously improving the infrastructure construction of the game industry through the establishment of communities, talent training and etc.</p>
<p>It is precisely because of its years of in-depth exploration in all aspects of game development that UWA has long known the various pain points of the current industrialization development stage and the urgent needs of the developers.</p>
<p><a href="https://l.facebook.com/l.php?u=https%3A%2F%2Fblog.en.uwa4d.com%2F2021%2F12%2F08%2Fwith-35-lectures-in-2-days-focusing-on-industrialization-problems-what-substantial-knowledge-of-game-rd-did-uwa-day-2021-bring%2F%3Ffbclid%3DIwAR3W2xPMT0N652xsk31f5cJUwXQpFwEbQAIqEleaEXIExYHnQ_6QHAxJ_Vs&amp;h=AT1S9tQjPvEsnp5sG-cno_1aASjk1tcmJdGeufp4mr6tKLKkglKAaeJvVktTNCZ8OifxoMLHmlcxR_cllzgrd5T9JEDus-51UBKgXFTq1MShP5FE2ShPmR2oI9r7PFOtc1txqBBs0A&amp;__tn__=-UK-R&amp;c%5B0%5D=AT3ShiExc1NLMbYQYTCmlOnLbIyOrkZHvkpUphqpPCwlGpsU9SuvVlh4Swirpm7sRaFzr_aBCDeTCHX_DTMVOB1xg0sJ2YDFN7sB8teSgyvPTovdO4LeB4SV6p-pjECLdgshzWY8c_uW1wstkz8MmBa9eBv5HV9S0GWwHKVKvWwmqfDd8V5TsFiVNlqApy1PhBNXcLq5iFnTUszBmUQ" rel="noopener nofollow ugc">https://blog.en.uwa4d.com/…/with-35-lectures-in-2-days…/</a></p>
